client/v2/common/models/participation_updates.go

```diff
@@ -6,4 +6,8 @@ type ParticipationUpdates struct {
 	// ExpiredParticipationAccounts (partupdrmv) a list of online accounts that needs
 	// to be converted to offline since their participation key expired.
 	ExpiredParticipationAccounts []string `json:"expired-participation-accounts,omitempty"`
+
+	// AbsentParticipationAccounts contains a list of online accounts that
+	// needs to be converted to offline since they are not proposing.
+	AbsentParticipationAccounts []string `json:"absent-particpation-accounts,omitempty"`
```

Comment:
This file is generated, so you shouldn't manually change it

Reply:
Unfortunately we don't do a good job of broadcasting that it's generated. We really should add a header saying so. Most things under the `client/v2` folder are generated by code from https://github.com/algorand/generator

This particular model is from indexer: https://github.com/algorand/indexer/blob/5ce3a09cc9e55a523a6e085930674fa9ee14fe73/api/indexer.oas2.json#L1847

The order of operations that makes the most sense would probably be merging this PR without this change, updating the indexer API, then regenerating this SDK's models based on that. Saying this because indexer will need this SDK's updated BlockHeader struct in order to populate its API response

Reply:
If I have this right, adding a field to the block header looks something like this.

1. Commit to go-algorand, despite the type check error.
2. Fix the type check error making the same changes in to go-algorand-sdk, and copy (most of) consensus.go
3. Check that, by re-running CI on go-algorand
4. Make the same changes (? not sure yet) in indexer
5. Go back and finish the changes in go-algorand-sdk, probably copying the models from indexer?